ui/tests/integration/components/secret-engine/configure-aws-test.js

```diff
@@ -38,6 +38,8 @@ module('Integration | Component | SecretEngine/configure-aws', function (hooks)
     this.leaseConfig = this.store.createRecord('aws/lease-config');
     // Add backend to the configs because it's not on the testing snapshot (would come from url)
     this.rootConfig.backend = this.leaseConfig.backend = this.id;
+    this.version = this.owner.lookup('service:version');
```

Comment:
I think it makes sense to set the version within the `isEnterprise` and `isCommunity` modules so they reflect those scenarios